tutorials/connect-jdbc-specificavro/kstreams.html

```diff
@@ -1,14 +0,0 @@
----	
-permalink: /connect-jdbc-avro/kstreams
-canonical: connect-jdbc-kstreams-rekey-specificavro/kstreams.html
```

Comment:
We need to preserve this file so that existing links are redirected. Previous file content should be retained, with this change:
`canonical: connect-add-key-to-source/kstreams.html`

Reply:
@rmoff, sorry for not being clearer in my review that both deletions need to be reverted. 

With c8ea79f, one of the two files is reverted and looks good. But we still need to undelete this one with the change in the comment above, because this tutorial was renamed twice.

In other words, one of two suggestions from my previous review is outstanding:
- [x] `tutorials/connect-jdbc-kstreams-rekey-specificavro/kstreams.html` c8ea79f
- [ ] `tutorials/connect-jdbc-specificavro/kstreams.html`

Reply:
Ah my bad, didn't read carefully enough :) I'll fix it now.

Reply:
@colinhicks can you check this again - I think I've fixed it now.